README.md

```diff
@@ -6,58 +6,55 @@
 
 Fylla, the Norse word for `complete`, is an autocompletion script generator for the [Thor](whatisthor.com) framework.
 
-It currently generates zsh completion scripts, but will soon have support for bash scripts as well. 
+Fylla currently generates zsh completion scripts. Support for bash scripts will be added.
 
 ## Installation
 
-Add this line to your application's Gemfile:
+Add the following line to your application's Gemfile:
 
 ```ruby
 gem 'fylla'
 ```
 
-And then execute:
+execute using the following command:
 
     $ bundle
 
-Or install it yourself as:
+Or install fylla manually:
 
     $ gem install fylla
 
 ## Usage
 
-`Fylla` must be loaded before `Thor.start` is called in order for it to be used. 
+`Fylla` must be loaded before `Thor.start` can be used. 
 
 General use case will be to create a new subcommand or option that calls `Fylla.zsh_completion(self)`
 
-`Fylla#zsh_completion` returns a string containing the entire zsh completion script, that you can 
-use to do what you see fit.
+`Fylla#zsh_completion` returns a string containing the entire zsh completion script and bash completion script. Scripts are ready for use after `Fylla#zsh_completion` returns.
 
-Same for bash completion.
-
-The only requirement for calling `Fylla.zsh_completion(self)` is that `Thor` has loaded all commands/options/etc.
+`Thor` is required to load commands/options/etc before calling `Fylla.zsh_completion(self)`.
 
 ## Development
 
-After checking out the repo, run `bin/setup` to install dependencies. Then, run `rake test` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
+After checking out the repo, run `bin/setup` to install dependencies. Then, run `rake test` to run the tests. Run `bin/console` for an interactive prompt that allows experimentation.
 
```

Comment:
Everything from here to the Todo list is generated by ruby so it should probably be left alone. Maybe small steps 😜

Reply:
All changes after this part are very small. Sentences that start with 'This' is replaced with 'The project'. 'Welcome' changed to 'welcomed' as using 'welcomed' makes sense in the sentence. All changes are are small and mostly single words. Sentence structure is not changed. Edits are required in part generated by ruby...